specification/snapshot/create/SnapshotCreateRequest.ts

```diff
@@ -64,7 +64,7 @@ export interface Request extends RequestBase {
      */
     wait_for_completion?: boolean
   }
-  body: {
+  body?: {
```

Comment:
YAML test that creates a snapshot without a body: https://github.com/elastic/elasticsearch/blob/a82b5cfb0ff7d22b95c56a8e7dcd7c5f3ba4ff24/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/snapshot.create/10_basic.yml#L23-L27